widgets/alloy/nuxeo-alloy-editor.html

```diff
@@ -0,0 +1,126 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../nuxeo-html-editor.html">
+<link rel="import" href="alloy-import.html">
+<link rel="import" href="alloy-ocean.html">
+
+<script>
+  {
+    class Editor extends Nuxeo.HTMLEditor {
+
+      static get template() {
+        return Polymer.html`
+          <style include="alloy-ocean">
+            #editor {
+              outline: none;
+              height: 100%;
+              min-height: 30em;
+            }
+
+            .ae-placeholder:empty:not(:focus):before {
+              color: grey;
+              font-style: italic;
+            }
+
+            div#editor > * {
+              margin-top: 0;
+            }
+          </style>
+
+          <div id="editor" data-placeholder$="[[placeholder]]"></div>
+        `;
+      }
+
+      static get observers() {
+        return ['_valueChanged(value)'];
+      }
+
+      connectedCallback() {
+        super.connectedCallback();
+        setTimeout(() => {
+          this._init();
+        }, 100);
```

Comment:
Why do we need to wait for 100ms? Wouldn't a `microTask.run` be enough?

Reply:
tbh I haven't touched this code at all to avoid breaking things....

Reply:
Fair enough... 😬